vllm/entrypoints/openai/rpc/client.py

```diff
@@ -32,26 +33,33 @@ def close(self):
         """Destroy the ZeroMQ Context."""
         self.context.destroy()
 
-    async def _send_one_way_rpc_request(self, request: RPC_REQUEST_TYPE,
-                                        error_message: str):
-        """Send one-way RPC request to trigger an action."""
+    @contextmanager
```

Comment:
does this need to be `asynccontextmanager`?

Reply:
I don't think so because it doesn't use any async methods or do any blocking

Reply:
Based on this I think it's just poll / send / recv that need to be handled with async:

https://pyzmq.readthedocs.io/en/latest/api/zmq.asyncio.html#socket